### ISSUE TITLE
All sensors become unavailable after having enabled disabled sensors

### ISSUE THREAD

**Issue by KDYR** [wmbusmeters/wmbusmeters#924]
### Type of request

Bug

### HAOS version

Home Assistant 2023.3.2 Supervisor 2023.03.1 Operating System 9.5 Frontend 20230306.0

### wmbusmeters version

1.12.0

### Your message goes here

I have upgraden from previous wmbusmeters strain, and after having configured manually (there was no 'restore' as suggested in the guide) my Kamstrup FlowIQ 2200 shows up again.

Some sensors are disabled as default, and I tried to enable them applying usual procedure; I also got the "will be included in 30 sec" message. After that all sensors become unavailable. A restart of the Add-On did not help. Only a restart of HA fixed the issue: After that the sensors again become available. 

In the Add-On "documentation tab - I get this error "Failed to get add-on documentation, Unknown error, see supervisor logs". However there is nothing about wmbusmeters in the Superwisor log, and there are no errors in the Add-On log itself (not even in 'debug' level)

In summary: 

- No 'Restore' available in configuration
- Sensors become unavailable
- Documentation tab empty

**Comment by KDYR** [comment]
Thx for looking into this!

1. I installed the stable version. 
2. Not a big issue, but yes, the phrasing let me to think there was a function. But manual configuration based on my downloaded txt-version of the yaml, was easy.
3. I waited 5 mins+, I think.

BR

**Comment by BIBOLV** [comment]
- Updated docs to point out explicitly that those are manual actions
- After new data received, sensors should change state to available, but depending on reporting frequency it could take longer time. Let`s skip this for now - until there are more reports about such behavior. 
- I installed stable version, checked everything and all is good with docs. Do you still have issue? Maybe try to reinstall addon?

**Comment by KDYR** [comment]
THX
